src/action.js

```diff
@@ -81,35 +82,33 @@ async function GetLatest() {
         throw new Error(err)
     })
 
+    ProcessRelease(latestRelease)
+
+    core.info('GetLatestRelease Done')
+}
+
+async function ProcessRelease(release) {
     const {
-        data: { tag_name: latestTag, name: latestName, body: latestBody }
-    } = latestRelease
-    latestAsset = latestRelease.data.assets || ''
-    core.info(
-        format('latest_tag:%s,latest_name:%s,latest_body:%s', latestTag, latestName, latestBody))
-
-    tag = latestTag
-    if (body != '') {
-        body += latestBody != '' ? "\n" + latestBody : ''
-    } else {
-        body = latestBody
-    }
-    if (releaseName != '') {
-        releaseName += latestName != '' ? " " + latestName : ''
-    }
-    core.info('GetLatest Done')
+        data: { tag_name: t, name: n, body: b }
+    } = release
+    releaseAsset = release.data.assets || ''
```

Comment:
this line (releaseAsset = release.data.assets || '') is "clever" (e.g. what type is releaseAsset?  where is it used? where is it declared)...I know you want "minimal changes", but this might be worth clarifying.